lib/network/kmip/Client.ts

```diff
@@ -302,11 +308,12 @@ export default class Client {
      * Activate a cryptographic key managed by the server,
      * for a specific bucket. This is a required action to perform after
      * the key creation.
-     * @param keyIdentifier - The bucket key Id
+     * @param keyIdentifierOrArn - The bucket key Id
      * @param logger - Werelog logger object
      * @param cb - The callback(err: Error)
```

Comment:
minor: Maybe also update `callback(err: Error)`.

Same in `createBucketKey`.

Reply:
Fixed